packages/flutter/lib/src/animation/animation_controller.dart

```diff
@@ -335,6 +339,14 @@ class AnimationController extends Animation<double>
       final double range = upperBound - lowerBound;
       final double remainingFraction = range.isFinite ? (target - _value).abs() / range : 1.0;
       simulationDuration = this.duration * remainingFraction;
+    } else if (target < _value + _kAnimateToTolerance.distance && target > _value - _kAnimateToTolerance.distance) {
```

Comment:
We have a method that wraps this expression, I think. `nearEqual` or some such.
(Might be worth putting it on Tolerance itself...)